### PR TITLE
Add ARM Embedded GCC toolchain to whitelist #132

### PR DESCRIPTION
**Pull request by kaidokert**
ARM toolchains have been asked multiple times as per #132

### CODE REVIEW

File: ubuntu.json [hunk 1]
```diff
@@ -270,6 +270,11 @@
     "key_url": "https://www.rabbitmq.com/rabbitmq-signing-key-public.asc"
   },
   {
+    "alias" : "team-gcc-arm-embedded",
```
Comment (alevy) [inline ubuntu.json]: It looks like removing the spaces before colons would satisfy the test suite

Comment (kaidokert) [inline ubuntu.json]: Probably right - removed, thanks. I forgot about this one as i figured they wont be adding new sources anyway for the time being

**Comment by fnoble** [comment]
+1 really need this

Tests are passing, can we get this in? Issue was open for 6 months now...

**Comment by alevy** [comment]
@BanzaiMan is this blocking on anything @kaidokert (or someone else) can fix? The Ubuntu official version of arm-none-eabi is very behind, and this apt repository is basically the gold standard (AFAIK) for embedded development. It would be a huge benefit to be able to use this with the new infrastructure.

**Comment by kaidokert** [comment]
Per travis-ci/docs-travis-ci-com/pull/390 , new approval requests do not seem to be currently processed at all.

**Comment by alevy** [comment]
Except there are merged requests as of a few days a go. This is pretty confusing :disappointed:

**Comment by elcojacobs** [comment]
+1. Is it possible to use gcc-arm-embedded on the container infrastructure without this being approved? This our only command that requires sudo.

**Comment by cgundogan** [comment]
also +1 here. Would be nice to have this finally in. The switch to the container based infra would be a blessing as the other infras are flat-lining currently..

**Comment by wilberforce** [comment]
@MariadeAnton hi - any chance of getting this committed? Thanks.

**Comment by wilberforce** [comment]
@BanzaiMan Please can we get the arm toolchain added?

**Comment by tannewt** [comment]
@kaidokert was this closed because none are being added or because it was added? Thanks!

**Comment by kaidokert** [comment]
None are being added. I'm using something like this to pull down toolchains :

https://github.com/kaidokert/mynewt-blinky/blob/master/ci/_unfinished_install.sh

**Pull request closed without merging**